pygenn/genn_groups.py

```diff
@@ -632,6 +632,18 @@ def get_sparse_post_inds(self):
                 raise Exception("only manually initialised connectivity "
                                 "can currently by accessed")
 
+            if self.connectivity_initialiser is not None:
+                # download the connectivity details from device
+                self.pull_connectivity_from_device()
+
+                # the ind array still has some non-valid data so we remove them
+                # with the row_lengths data
+                ind = [self.ind[start_idx + j]
+                        for i, start_idx in enumerate(xrange(0, len(self.ind), self.max_row_length))
+                            for j in range(self.row_lengths[i])]
```

Comment:
Doing this in python will be mega slow and memory-eating as these arrays can be gigabytes. Maybe try some numpy like:

```python
return np.hstack([self._ind[i * self._max_row_length, (i * self._max_row_length) + r] 
                  for i, r in enumerate(self._row_lengths)])
```